### ISSUE TITLE
Maintenance status of dbt2looker

### ISSUE THREAD

**Issue by PaddyAlton** [lightdash/dbt2looker#100]
I am writing this issue to query what the current maintenance status of this project is, and what the plans are (if any) for its future. At the moment the project (in its default state) is broken due to dependency issues.

I have found it to be a useful project. We use it regularly. However, I've been running a custom fork of it since January - and while I have made a series of PRs this year, I haven't heard back about them (the first received an initial review but no follow up).

I do appreciate that this isn't core business for Lightdash and that you've provided this useful tool for free. I'm sure you want to limit any time spent working on it. That being so, I have some suggestions:

- there may be some appetite in the community to do as I have done and make contributions, or even for community maintainers to be included so that progress can be made on some of the open issues without requiring time from Lightdash
- alternatively it may be that you want to officially deprecate the tool and mark it clearly as unsuitable for use in new projects

I think either of these would be acceptable - I think the community would welcome the clarity in any case.

Many thanks in advance.

**Comment by tobiascadee** [comment]
@PaddyAlton I would love to use this project, is your fork public? Also big up for this, and I would definitely be up to contribute to this project.

**Comment by PaddyAlton** [comment]
[Fork is here](https://github.com/PaddyAlton/dbt2looker), should be public @tobiascadee .

If you check out `97-fix-pydantic-import`, that is the tip of the stack of open PRs.

**Comment by tobiascadee** [comment]
amazing, if you don't get a reply here maybe its time to start a  new maintained repo

**Comment by PaddyAlton** [comment]
Agree. But to be honest - the difficulty of integrating Looker with DBT (among other things) is pushing me away from Looker. I may not use it much longer.

If I do keep using it, and if `dbt2looker` is defunct, I will probably take a closer looker at [optician](https://github.com/getground/optician), which I came across a few months back. Might be better to collaborate on an active alternative before moving to set up a separate fork of `dbt2looker` on PyPI.

**Comment by tobiascadee** [comment]
For me moving away from Looker is not an option unfortunately and I feel that just creating lookml from the database definition would be insufficient for me.

**Comment by charlesphil** [comment]
Just wanted to chime in here that I found @PaddyAlton's PR https://github.com/lightdash/dbt2looker/pull/98 useful and would love to see it merged in.

That being said, it does seem like Lightdash integrated "`dbt to lookml`" (and the opposite) in their main product (https://github.com/lightdash/lightdash/issues/262) and it seems unlikely to me that they'll revisit this project. I'd also love to hear from maintainers as to what the status of `dbt2looker` is on their end and if we should fork or move away to another project entirely.

**Comment by PaddyAlton** [comment]
Tagging @owlas and @TuringLovesDeathMetal as the two main contributors, just so as to draw this thread to their attention. If you've got a moment to let us know how things stand I'm certain it would be much appreciated.
